pages/location.jsx

```diff
@@ -4,7 +4,7 @@ var Footer = require('../components/footer.jsx');
 var ImageTag = require('../components/imagetag.jsx');
 var generateHelmet = require('../lib/helmet.jsx');
 
-var accessToken = 'pk.eyJ1IjoiYWxpY29kaW5nIiwiYSI6Il90WlNFdE0ifQ.QGGdXGA_2QH-6ujyZE2oSg';
+var accessToken = 'pk.eyJ1IjoibW9mby1kZXZvcHMiLCJhIjoiY2l2MmR5OWdpMDFrbzJvb2VteGZhaG9oZCJ9.GdlQ7Bq3NOjlomZsK2IX0w';
```

Comment:
Here I replaced the outdated token (not sure where we got that token from) to the public token that's listed on mofo devops mapbox account.